compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/uikit/ComposeUIViewControllerConfiguration.uikit.kt

```diff
@@ -66,6 +68,15 @@ class ComposeUIViewControllerConfiguration {
      * explanation on how to fix the issue.
      */
     var enforceStrictPlistSanityCheck: Boolean = true
+
+    /**
+     * Container view for additional compose layers, used to display full screen overlay interface
+     * elements such as popups or text selection handlers The container view should bb placed on the
+     * same UIWindow as the [ComposeUIViewController].
+     * If not set, the containing UIWindow of the [ComposeUIViewController] is used.
+     */
+    @ExperimentalComposeApi
+    var windowContainerView: UIView? = null
```

Comment:
Currently, I'm not sure that we really need this. Could you please emphasize reasoning and use cases?